.gitignore

```diff
@@ -5,3 +5,4 @@ dist
 .opt-in
 .opt-out
 npm-debug.log
+.idea/
```

Comment:
Should probably be in your global gitignore file ;-) But it's fine. I don't mind adding it.